src/components/activity-page/managed-interactive/managed-interactive.tsx

```diff
@@ -34,7 +36,7 @@ const getModalContainer = (): HTMLElement => {
   return document.getElementById("app") || document.body;
 };
 
-export const ManagedInteractive: React.FC<IProps> = (props) => {
+export const ManagedInteractive: React.FC<IProps> = observer(({ ...props }) => {
```

Comment:
It's not clear to me what we're observing here. I believe `pluginInfo` is the only MobX observable around, but it doesn't appear to be referenced explicitly below. It is referenced by the `<IframeRuntime>` component, but that wouldn't explain why this component needs to rerender when it changes. If this is necessary for some non-obvious reason a comment would be useful.

Reply:
I thought that got ripped out when we worked on the observable - must have been overlooked.

Reply:
Fixed.